ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/IntroEligibilityStateView.kt

```diff
@@ -33,14 +35,20 @@ internal fun IntroEligibilityStateView(
         ?: textWithIntroOffer
         ?: ""
 
-    Markdown(
-        text,
-        color = color,
-        style = style,
-        fontWeight = fontWeight,
-        textAlign = textAlign,
-        modifier = modifier,
-    )
+    Crossfade(
```

Comment:
I was wondering whether we should optimize and only apply the Crossfade in templates that allow to select a package... But I think it should be ok since we're not changing the text anyway in those.